plugins/logitech-bulkcontroller/fu-logitech-bulkcontroller-common.h

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright (c) 1999-2021 Logitech, Inc.
+ * All Rights Reserved
+ *
+ * SPDX-License-Identifier: LGPL-2.1+
+ */
+
+#pragma once
+
+#include <glib.h>
+
+#include "usb_msg.pb-c.h"
+
+typedef enum {
+	kDeviceStateUnknown = -1,
+	kDeviceStateOffline,
+	kDeviceStateOnline,
+	kDeviceStateIdle,
+	kDeviceStateInUse,
+	kDeviceStateAudioOnly,
+	kDeviceStateEnumerating
+} FuLogitechBulkcontrollerDeviceStatus;
+
+typedef enum {
+	kUpdateStateUnknown = -1,
+	kUpdateStateCurrent,
+	kUpdateStateAvailable,
+	kUpdateStateStarting = 3,
```

Comment:
are we missing a device update state?

Reply:
Sorry for the delay. I was off last week. We have kUpdateStateUpdating

Reply:
Added it as a fixup commit, can squash when it's all working, or drop if you decide this shouldn't be in the code.